README.md

```diff
@@ -106,6 +106,43 @@ bundle exec rake
 
 To run the feature specs you will need Chrome installed.
 
+### Running a live console
+
+**Accessing a live console is very risky and should only be done as a last
+resort. This should only be done in pairs, and mutating any live data is
+STRONGLY discouraged.**
+
+The console will be ran inside a container instance and won't be on one of the
+web servers, however it will have access to the database.
+
+#### Through the Azure UI
+
+- Navigate to the container instance resource (eg. `s118d01-app-worker-aci`)
+- Then go to 'Containers' under 'Settings'
+- With the container selected go to the 'Connect' tab
+- Choose the start up command (`/bin/bash` is recommended) and connect
+
+#### Through the Azure CLI
+
+We have a helpful script you can run that will connect you to the right
+resource:
+
+```bash
+bin/azure-console $ENVIRONMENT # (development/production)
+```
+
+#### Usage
+
+If you don't need to modify the data and only need to do queries it is
+recommended that you run rails console in sandbox mode
+`rails console --sandbox`.
+
+In exceptional circumstances you may want to modify the data, this should only
+be done with approval from the Service Owner and is to be carried out in pairs.
+
+Accessing a live console on production requires a PIM (Privileged Identity
```

Comment:
Do we have any documentation about how to do this? I feel like I saw some in some other context, but I can't remember where.

Reply:
Yeah I feel like I've seen it on another page but I can't find where. I've created a separate page on Confluence for it and I'll link to it from the readme